packages/saltcorn-data/models/file.ts

```diff
@@ -156,21 +156,37 @@ class File {
     const s = absPath.replace(path.join(db.connectObj.file_store, tenant), "");
     return s[0] === "/" ? s.substring(1) : s;
   }
+
+  private static dirCache: File[] | null = null;
```

Comment:
is this ok on node 14? what about race conditions on multitenant systems? 

suggestion: move to an object at the top level of the module (for node 14); key in object should be tenant name

Reply:
I've added an 'ignoreCache' option. Or Would you like it the other way around with an useCache option?

But I believe then, modules with fieldviews like that

https://github.com/saltcorn/ckeditor4/blob/cee08465a44441765815626d9c8357e8497a9406/index.js#L35

need to add true.